packages/manager/tools/component-rollup-config/package.json

```diff
@@ -62,14 +50,12 @@
     "slash": "^2.0.0"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.11.0",
-    "@typescript-eslint/parser": "^2.11.0",
+    "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0",
```

Comment:
dev dependency of itself ?

Reply:
Nice, should be removed, a lousy copy/paste i forgot !